v/index.md

```diff
@@ -0,0 +1,8 @@
+# List of artists starting with V
+
+- [Vaiavy chilla](vaiavy-chilla.md)
+- [Vivy](vivy.md)
+- [Voahangy](voangy.md)
+- [Voary](voary.md)
+
+[Go back to the artist index](http://../index.md)
```

Comment:
This link needs to have the `http://` removed as we just want to use relative URLs.